runtime/doc/lua.txt

```diff
@@ -2680,6 +2707,21 @@ vim.ui.select({items}, {opts}, {on_choice})                  *vim.ui.select()*
                      made a choice. `idx` is the 1-based index of `item`
                      within `items`. `nil` if the user aborted the dialog.
 
+vim.ui.spinner({steps}, {on_step}, {opts})                  *vim.ui.spinner()*
+    todo
+
+    Parameters: ~
+      • {steps}    (`vim.ui.SpinnerStep[]?`) See |vim.ui.SpinnerStep|.
+      • {on_step}  (`fun(char: string, char_id: integer)?`) Callback to be
+                   executed on every step update. Can include forcing
+                   `vim.cmd.redrawstatus()`, etc.
+      • {opts}     (`table?`) Additional options.
+                   • {reverse} (`boolean`) Whether to reverse from last to
```

Comment:
I now don't think `reverse` is a necessary option to have here. It can be relatively straightforwardly emulated by the caller with adjusting `steps`.